source/client/output_formatter_impl.cc

```diff
@@ -288,45 +287,54 @@ std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Outp
   fortio_output.set_numthreads(number_of_connections);
 
   // Get the result that represents all workers (global)
-  const auto& nh_global_result = this->getGlobalResult(output);
+  const auto& nh_global_result = getGlobalResult(output);
 
   // Fill in the actual QPS based on the counters
-  const auto& nh_rq_counter = this->getCounterByName(nh_global_result, "upstream_rq_total");
-  const double actual_qps = static_cast<double>(
-      nh_rq_counter.value() / std::chrono::duration<double>(actual_duration).count());
-  fortio_output.set_actualqps(actual_qps);
+  if (!visitCounter(
+          nh_global_result, "upstream_rq_total",
+          [&fortio_output, actual_duration](const nighthawk::client::Counter& nh_counter) {
+            const double actual_qps = static_cast<double>(
+                nh_counter.value() / std::chrono::duration<double>(actual_duration).count());
+            fortio_output.set_actualqps(actual_qps);
+          })) {
+    throw NighthawkException(
+        "Nighthawk result was malformed, contains no counter with name: upstream_rq_total");
+  };
+  visitCounter(nh_global_result, "upstream_cx_rx_bytes_total",
```

Comment:
Is the intended behavior that `upstream_cx_rx_bytes_total` and `upstream_cx_tx_bytes_total` will be unset in the fortio output if those counters were missing?

Reply:
> Is the intended behavior that `upstream_cx_rx_bytes_total` and `upstream_cx_tx_bytes_total` will be unset in the fortio output if those counters were missing?

No, thinking about it, it would probably be helpful to consumers to zero them. Thanks for catching that.